src/main/java/com/minecolonies/coremod/entity/citizen/citizenhandlers/CitizenSkillHandler.java

```diff
@@ -180,7 +177,7 @@ public void addXpToSkill(final Skill skill, final double xp, final ICitizenData
 
         final IBuilding home = data.getHomeBuilding();
 
-        final double citizenHutLevel = home == null ? 0 : home.getBuildingLevel();
+        final double citizenHutLevel = home == null ? 1 : home.getBuildingLevel();
```

Comment:
Why treat homeless people as if they have a level 1 house?

Reply:
I've raised the base here, because before homeless people never could gain any XP. However now that there is a chance to give them some I think having 1 be the lowest makes sense

Reply:
Then why not just +1 the level (and maxlevel), such that people in a L1 house do still get a skillcap boost over L0?  Pushing L4 to a cap of 50 (from 40) doesn't seem like a problematic change, there's still lots of headroom before L5.

On an unrelated note (existing prior to this PR), it looks like a citizen living in an L3 tavern will have no skill cap, just like one living in an L5 house.  Is that intended?

Reply:
ye added +1, disallowed all buildings below max 5 from going up higher